### DISCUSSION TITLE
HEXO DEPLOY will not change GitHub Page‘s file structure

### DISCUSSION THREAD

**Discussion by Korltex** [hexojs/hexo#5443] [Question / Answer] [answered]
Today I just change my post path，like : `name.github.io/2024/03/27/HalloWorld/index.html`. After `hexo d`, I fond GithHub's post path still : `name.github.io/2024/03/27/halloworld/index.html`. At the same time,  the content path of `index.html` changed to : `/HalloWorld/ `. That result in a 404 error.

Is it true that I can't modify the path at will?

**Comment by stevenjoezhang** [comment, marked as answer]
If the file system is case-insensitive, `git` may not update the filename correctly. You have to remove it from `git` and then re-add it.

**Reply by Korltex** [reply]
Thank you，that's a nice solution. I will give a try.
